### ISSUE TITLE
Multi-file upload from host root results in a bad filename

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#8399]
If you try to use the multi-file upload from the host root folder and upload any file (an image, for example), you end up with a file with a name based on the tmp file,  something like:

/upload_00000017.png

This is true for 3.2.4 as well as 3.3

![screen shot 2015-11-30 at 5 24 08 pm](https://cloud.githubusercontent.com/assets/934364/11486450/38a9d006-9787-11e5-8d9f-07aa75f9016f.png)

You can see this in the identifier table as well

```
mysql> select * from identifier where id='0eb3a6a8-05d5-4c95-a338-b673c7df8c82';
+--------------------------------------+-------------+----------------------+--------------------------------------+------------+-----------------+----------------+
| id                                   | parent_path | asset_name           | host_inode                           | asset_type | syspublish_date | sysexpire_date |
+--------------------------------------+-------------+----------------------+--------------------------------------+------------+-----------------+----------------+
| 0eb3a6a8-05d5-4c95-a338-b673c7df8c82 | /           | upload_00000017.html | 48190c8c-42c4-46af-8d1a-0cd5db894797 | contentlet | NULL            | NULL           |
+--------------------------------------+-------------+----------------------+--------------------------------------+------------+-----------------+----------------+
1 row in set (0.00 sec)
```

**Comment by brentgriffin** [comment]
Added priority because causing issue with naming of attachments in support portal - https://github.com/dotCMS/SupportPortal/issues/193

**Comment by oswaldogallango** [comment]
The issue reported in this ticket was solved on 3.6. Here are the lines that fix the problem  https://github.com/dotCMS/core/blob/master-3.6/src/com/dotmarketing/portlets/files/action/UploadMultipleFilesAction.java#L271-L273. The only thing is that all the old file assets will remain with the upload_xxx.yy filename, only the new ones will be saved with the correct filename
